CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Changed
+
+- Tooling has been updated to use .NET Core SDK v3.1.3xx. [#287](https://github.com/spatialos/gdk-for-unity-fps-starter-project/pull/287)
+
```

Comment:
My opinion is that we can generally omit these as it will be captured in the 'Updated to GDK version x.y.z.' that's inserted by the release tool

Reply:
+1 since the change wasn't in in the FPS/Blank project themselves